blocks/library/pullquote/editor.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 .wp-block-pullquote {
-	footer .blocks-editable__tinymce[data-is-empty="true"]:before {
```

Comment:
Is there ever a scenario where this rule is applied? Looks like this is old and safe to remove.

Reply:
Just tell me (anyone) if I should remove it, or not. 🙂